lib/Ast.ml

```diff
@@ -321,6 +321,8 @@ module Exp = struct
     | Pexp_array [x] | Pexp_list [x] -> is_trivial x
     | _ -> false
 
+  let is_simple = is_trivial
```

Comment:
There's a `is_simple` already in the module `Requires_sub_terms` but it is a lot less restricted, for example a list of trivial elements is simple. I think the wrap-or-vertical mode should use `is_trivial`.

Reply:
done